app/admin/widgets/Filter.php

```diff
@@ -151,6 +157,11 @@ public function onSubmit()
                     $this->setScopeValue($scope, $active);
                     break;
 
+                case 'selectlist':
```

Comment:
You can combine this case and the select case

Reply:
Changed